specification/app/data-plane/Microsoft.App.DynamicSessions/preview/2024-10-02-preview/DynamicSessions.json

```diff
@@ -0,0 +1,759 @@
+        },
+        "stdout": {
+          "type": "string",
+          "description": "The standard output of the code execution."
+        },
+        "stderr": {
+          "type": "string",
+          "description": "The standard error of the code execution."
+        },
+        "result": {
+          "description": "The result of the code execution. The type of this field is same as the type of actual result of the code execution after being Json serialized."
+        },
+        "executionTimeInMilliseconds": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The execution time of the code in milliseconds."
+        }
+      },
+      "required": [
+        "id",
+        "status",
+        "stdout",
+        "stderr",
+        "result",
+        "executionTimeInMilliseconds"
+      ]
+    },
+    "SessionResourceFile": {
+      "type": "object",
+      "description": "Code execution file resource.",
+      "properties": {
+        "fileName": {
+          "type": "string",
+          "description": "The name of the file.",
+          "x-ms-mutability": [
+            "read",
+            "update",
+            "create"
+          ]
+        },
+        "sizeInBytes": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The size of the file."
+        },
+        "createdAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The date time in RFC3339 format when the file was uploaded."
+        },
+        "lastModifiedAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The date time in RFC3339 format when the file was last modified."
+        }
+      },
+      "required": [
+        "fileName",
+        "sizeInBytes",
+        "createdAt",
+        "lastModifiedAt"
+      ]
+    },
+    "SessionResourceFileCollection": {
+      "type": "object",
+      "description": "Code execution file resource collection.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "The files uploaded.",
+          "items": {
+            "$ref": "#/definitions/SessionResourceFile"
+          },
+          "x-ms-identifiers": []
+        }
+      },
+      "required": [
+        "value"
+      ]
+    }
+  },
+  "parameters": {
+    "Azure.Core.Foundations.ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "description": "The API version to use for this operation.",
+      "required": true,
+      "type": "string",
+      "minLength": 1,
+      "x-ms-parameter-location": "method",
+      "x-ms-client-name": "apiVersion"
+    },
+    "CodeExecutionIdentifier": {
+      "name": "executionId",
+      "in": "path",
+      "description": "The identifier of code execution.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "SessionIdentifier": {
+      "name": "identifier",
```

Comment:
I think this name should be more descriptive.
```
      "name": "sessionId",
```

Reply:
Yes, make sense, updated.

Reply:
Please consider making this change before this service moves to GA.